gulpfile.js

```diff
@@ -5,19 +5,28 @@
 // *************************************
 //
 // Available tasks:
-//   `gulp`
 //   `gulp css`
+//   `gulp lint-scss``
+//   `gulp importChangelogFile`

```

Comment:
Why not `import-changelog-file`? I don't like the mix of lowercase and camelCasing in the function names. I think we should follow a pattern and stick to lowercase function names.

Reply:
I agree, I will do some gulp cleanup next week in a seperate branch.